addon/components/nrg-radio.js

```diff
@@ -3,7 +3,7 @@ import NrgValidationComponent from './nrg-validation-component';
 
 export default class NrgRadioComponent extends NrgValidationComponent {
   get checked() {
-    return this.args.value && this.args.value == this.value;
+    return this.args.value ?? this.args.value == this.value;
```

Comment:
This one needs to be changed back.
In the nullish example, any truthy value returns true.
In the previous example, any truthy value is then checked for equality.
```suggestion
    return this.args.value && this.args.value == this.value;
```

Reply:
I thought I changed this one back, my bad. Can it be checked for strict equality?
```suggestion
    return this.args.value == this.value;
```

Reply:
The reason the first segment is there was to prevent the following scenario:
Both values being undefined causes the radio button to be filled in.

This scenario is unlikely to happen, though it might be possible if the radio button values are retrieved asynchronously.